Index: policykit-1-0.105.9/deepin/deepin_security_verify.whitelist
===================================================================
--- /dev/null
+++ policykit-1-0.105.9/deepin/deepin_security_verify.whitelist
@@ -0,0 +1,17 @@
+/usr/sbin/gparted
+/sbin/gparted
+/usr/bin/uos-apt-login
+/usr/bin/deepin-devicemanager
+/usr/bin/deepin-devicemanager-authenticateProxy
+/usr/bin/dfont-install
+/usr/bin/dfont-uninstall
+/usr/bin/logViewerAuth
+/usr/bin/deepin-graphics-driver-manager
+/usr/sbin/overlayroot-chroot
+/usr/sbin/overlayroot-disable
+/usr/sbin/overlayroot-enable
+/usr/bin/renice
+/usr/lib/dde-control-center/develop-tool
+/bin/restore-tool
+/usr/bin/kill
+/usr/lib/deepin-recovery/prepare_recovery
